test/parallel/test-module-loading-globalpaths.js

```diff
@@ -33,7 +34,7 @@ if (process.argv[2] === 'child') {
     assert.strictEqual(child.trim(), expectedString);
   };
 
-  const testFixturesDir = path.join(common.fixturesDir,
+  const testFixturesDir = path.join(fixtures.fixturesDir,
                                     path.basename(__filename, '.js'));
```

Comment:
Hey @JustinBeckwith! this is good but can be made even better using `fixtures.path()`... e.g.

```js
const textFixturesDir = fixtures.path(path.basename(__filename, '.js'));
```